src/content/development/contribution-guide/_index.en.md

```diff
@@ -91,3 +123,12 @@ To be expanded
 ### Test Days
 
 To be expanded
+
+[Backlog board]: #backlog-board
+[bugs]: #bugs
+[enhancements]: https://github.com/submariner-io/enhancements/issues
+[epic template]: https://github.com/submariner-io/enhancements/blob/devel/.github/ISSUE_TEMPLATE/epic.md
+[epic]: #epics
```

Comment:
This may be a sign that we're linking to the same things too much. Generally, we only need to link the first time something is mentioned in a doc or major-section.